steward/src/allocation.rs

```diff
@@ -24,17 +30,31 @@ pub fn bytes_to_hex_str(bytes: &[u8]) -> String {
         .fold(String::new(), |acc, x| acc + &x)
 }
 
+/// Struct TickWeights for time independent bollinger ranges
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub struct TickWeight {
+    pub upper: i32,
+    pub lower: i32,
+    pub weight: u32,
+}
+
+pub fn from_tick_weight(tick_weight: TickWeight) -> CellarTickInfo {
+    CellarTickInfo {
+        token_id: U256::zero(),
```

Comment:
Should this be zero when we end up making the contract call? It isn't being updated after the `CellarTickInfo` is created from the incoming `TickWeight` structs.

Reply:
I don't really know, this is what I saw in Zaki's former codes. What do you suggest?

Reply:
Hrmmm, this depends on how the data is being handled by the target contract's `rebalance` function.  Would have to look into it.